src/pykeen/pipeline/api.py

```diff
@@ -1000,8 +1000,8 @@ def pipeline(  # noqa: C901
         mapped_triples = validation.mapped_triples
 
     # Evaluate
-    # Reuse optimal evaluation parameters from training if available
-    if evaluator_instance.batch_size is not None or evaluator_instance.slice_size is not None:
+    # Reuse optimal evaluation parameters from training if available, only if the validation triples are used again
```

Comment:
when this says training, it's because it runs a fake "training" with the mapped triples for the evaluation data?

Reply:
No

Reply:
Oh, I see what's going on then